### PR TITLE
Update to ethereumjs-common v1.1.0

### PR DESCRIPTION
**Pull request by danjm**
This PR updates code and package.json to use ethereumjs-common v1.1.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "async": "^2.6.1",
     "ethashjs": "~0.0.7",
     "ethereumjs-block": "~2.1.0",
-    "ethereumjs-common": "~0.6.0",
+    "ethereumjs-common": "~1.0.0",
```
Comment (whymarrh) [inline package.json]: We should change this to a caret to allow for new minor versions (should be safe)<sup>[\[1\]](https://stackoverflow.com/a/22345808/1267663)</sup>

```suggestion
    "ethereumjs-common": "^1.0.0",
```

Comment (danjm) [reply package.json]: Thanks. Fixed.

**Comment by danjm** [comment]
@holgerd77 Updated ethereumjs-common to `1.1.0` and ethereumjs-block to `2.2.0`

**Comment by holgerd77** [comment]
Retriggered both builds here after [v2.2.0](https://github.com/ethereumjs/ethereumjs-block/releases/tag/v2.2.0) release of the block library.

**Pull request merged**